contrib/docs-examples/snippets/react-native/GoogleCredential.js

```diff
@@ -0,0 +1,98 @@
+function example() {} function untested({View, Text, Button, React, GoogleSignin, Stitch, GoogleCredential, GoogleSigninButton}) { // too much boilerplate to be able to test this
+  // Example component that uses the react-native-google-signin module to get
+  // the server auth code for Stitch to use.
+  //
+  // NOTE: The react-native-google-signin native module must be installed correctly
+  // and your Google project must be configured.
+  //
+  // For more detail, see https://stackoverflow.com/questions/55145071/#55173164
+  //
+  // Above:
+  // import {Stitch, GoogleCredential} from 'mongodb-stitch-react-native-sdk'
+  // import {GoogleSignin, GoogleSigninButton} from 'react-native-google-signin'
+  class GoogleLogin extends React.Component {
+    // ...
+
+    componentDidMount() {
+      // Configure react-native-google-signin
+      GoogleSignin.configure({
+        webClientId: '<id>', // client ID of type WEB from Stitch
+        offlineAccess: true, // allows Stitch service to use credential
+        iosClientId: '<id>', // [iOS] CLIENT_ID from GoogleService-Info.plist
+      })
+    }
+
+    _onPressLogin = async () => {
+      // It's recommended to call this before signIn()
+      await GoogleSignin.hasPlayServices()
+
+      // Sign in via react-native-google-signin
+      const userInfo = await GoogleSignin.signIn()
+
+      // Retrieve the server auth code
+      const {serverAuthCode} = userInfo
+      if (serverAuthCode === null) {
+        throw new Error('Failed to get serverAuthCode!')
```

Comment:
Does `serverAuthCode === null` imply that the login failed or was cancelled? Could add a comment clarifying the case that would cause the error.